src/main/java/xRoss/task/Deadline.java

```diff
@@ -0,0 +1,63 @@
+package xRoss.task;
+
+import xRoss.exception.EmptyStringException;
+
+/**
+ * Represents tasks with deadlines.
+ */
+public class Deadline extends Task {
+
+    /**
+     * by   Deadline date/time
```

Comment:
I think you should decide whether you want to have date or time or both as the deadline. That would help in streamlining the inputs.